rulesets/src/naming.ruleset.yml

```diff
@@ -16,9 +16,9 @@ rules:
     formats: [oas3]
     given: "$..properties.*~"
     then:
-      function: casing
+      function: pattern
       functionOptions:
-        type: camel
+        match: ^[a-z0-9]+(([A-Z]{2}|[A-Z])[a-z0-9]+)*$
```

Comment:
@travisgosselin as an aside, I don't see any rules about starting a property name with a number. Is that something we want to add to the rules? I assume it's not something we want to allow in the future

Reply:
Fixed this